test/e2e/features/pageObjects/headerPage.js

```diff
@@ -93,7 +95,13 @@ function HeaderPage() {
 
 
   this.getTabElementWithText = function (tabText) {
-    return this.primaryNavBar.element(by.xpath('//a[contains(text(),"' + tabText + '")]'));
+    return this.primaryNavBar.element(by.xpath('//a[contains(text(),"' + tabText + '")]'));;
```

Comment:
Change shown here because of two semi-colons - should get rid of one really

Reply:
@sreekanthpuligadda  can you do this as a separate change. This is not a blocker.